app/dashboard/templates/profiles/profile.html

```diff
@@ -262,17 +174,79 @@
 
     <script type="text/javascript">
       $(document).ready(function() {
-        if ($('.profile-bounties--tips .bounty_row--hidden').length > 0) {
-          var $button = $('.profile-bounties--tips .profile-bounties__btn-show-all');
-          $button.removeClass('hidden');
-          $button.on('click', function(event) {
-            $(this).remove();
-            $('.profile-bounties--tips .bounty_row').removeClass('bounty_row--hidden');
-          });
-        }
         setupTabs('#activity-tabs'); 
 
+        const tabSection = document.querySelector('#activity-tabs-sections');
```

Comment:
is it worth moving this out of the inline javascript and into a formal .js file?

Reply:
Because of the new functions we have now - Yes, big enough to move it to it's own file .

Reply:
yes pls